### PR TITLE
Minor improvements to /renameNpc

### PR DESCRIPTION
**Pull request by moisterrific**
Improved user feedback clarity and added silent specifier support.

?????? HAVE YOU UPDATED THE CHANGELOG? ??????
yes

### CODE REVIEW

**Comment by moisterrific** [comment]
Is it necessary to make the message more specific? 
Instead of `Player renamed the Guide.`, should I make it like `Player renamed Andrew the Guide to John the Guide.`?
Or `Player renamed the Guide to John.`?

Also is this supposed to only work on town NPCs? I was kind of confused at the conditions here:
https://github.com/Pryaxis/TShock/blob/356cdaa7ad710974791bdb9dd5fee9a2dfd477ab/TShockAPI/Commands.cs#L5803

File: TShockAPI/Commands.cs [hunk 1]
```diff
@@ -5798,22 +5798,31 @@ private static void RenameNPC(CommandArgs args)
...
 					done++;
+					npcType = Main.npc[i].TypeName;
 				}
 			}
 			if (done > 0)
 			{
-				TSPlayer.All.SendInfoMessage("{0} renamed the {1}.", args.Player.Name, args.Parameters[0]);
+				if (args.Silent)
+				{
+					TSPlayer.All.SendInfoMessage("Renamed the {0}.", npcType);
```
Comment (hakusaro) [inline TShockAPI/Commands.cs]: Can you not send a message if the command is silent?

**Review by hakusaro — COMMENTED** [review]
I think we should at least more or less try to be consistent: silent should mean silent imho.

**Comment by moisterrific** [comment]
> I think we should at least more or less try to be consistent: silent should mean silent imho.

Right, I'll change it soon.

Got confused at the time what silent was really supposed to do.

File: TShockAPI/Commands.cs (hunk 1 above)
Comment (moisterrific) [reply TShockAPI/Commands.cs]: just to make sure, silent = only the command user sees a success message?

Comment (hakusaro) [reply TShockAPI/Commands.cs]: Ya.

File: TShockAPI/Commands.cs [hunk 2]
```diff
@@ -5789,7 +5789,7 @@ private static void RenameNPC(CommandArgs args)
 				}
 				else if (args.Parameters[1].Length > 200)
```
Comment (steviegt6) [inline TShockAPI/Commands.cs]: Wouldn't this make the limit <= 200, not < 200?

Comment (bartico6) [reply TShockAPI/Commands.cs]: Correct, if it must be below 200, then the length test should be >= 200

Comment (moisterrific) [reply TShockAPI/Commands.cs]: yeah that makes sense, i somehow overlooked that lol

Comment (steviegt6) [reply TShockAPI/Commands.cs]: Also, is there any actual reason for the limit, or is it arbitrary?

Comment (moisterrific) [reply TShockAPI/Commands.cs]: I think it's to prevent crashing?

Comment (moisterrific) [reply TShockAPI/Commands.cs]: gonna comment out that check on my local build and see what happens

**Review by bartico6 — COMMENTED** [review]
Sincere apologies for late response - LGTM. I'm only iffy about the 200 character limit, as I'm not aware of any such limit in the game code, but to be honest, 200 characters sounds like a reasonable cap.

**And I misclicked, this is supposed to be an approving review, not a comment xD**

**Comment by moisterrific** [comment]
kinda confused on one thing, is this command only intended to be used on townNPCs?

**Pull request closed without merging**